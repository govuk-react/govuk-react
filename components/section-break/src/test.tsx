import * as React from 'react';
import { mount } from 'enzyme';

import { SectionBreak } from '.';

describe('SectionBreak', () => {
  it('renders without crashing', () => {
    mount(<SectionBreak />);
  });

  it('matches snapshot', () => {
    expect(mount(<SectionBreak visible level="LARGE" />)).toMatchSnapshot('section break');
  });
});
