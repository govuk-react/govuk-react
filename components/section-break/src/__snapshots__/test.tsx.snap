// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionBreak matches snapshot: section break 1`] = `
.c0 {
  margin: 0;
  border: 0;
  margin-top: 20px;
  margin-bottom: 20px;
  border-bottom: 1px solid #bfc1c3;
}

@media only screen and (min-width:641px) {
  .c0 {
    margin-top: 30px;
    margin-bottom: 30px;
  }
}

<SectionBreak
  level="LARGE"
  visible={true}
>
  <hr
    className="c0"
  />
</SectionBreak>
`;
